Traceback (most recent call last):
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\site-packages\nbclient\client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\site-packages\jupyter_core\utils\__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\asyncio\base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\pamcl\WPy64-31230\python-3.12.3.amd64\Lib\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from edcrop import edcrop
import os
import pandas as pd
import matplotlib.pyplot as plt
import warnings
warnings.simplefilter(action='ignore', category=FutureWarning)

wdir = os.path.join(os.path.dirname(os.getcwd()))

os.chdir(os.path.join(wdir, 'data/wetland/'))

yaml = 'edcrop.yaml'

edcrop.run_model(yaml=yaml)
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[1], line 1[0m
[1;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01medcrop[39;00m [38;5;28;01mimport[39;00m edcrop
[0;32m      2[0m [38;5;28;01mimport[39;00m [38;5;21;01mos[39;00m
[0;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mpandas[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpd[39;00m

[1;31mModuleNotFoundError[0m: No module named 'edcrop'

